[34;1mgetting data..[0m
[34;1m	-> done[0m
[32;1mStoring the output in: results/SynthText_8k.h5[0m
data/newsgroup/大提琴手高修.txt
data/newsgroup/小时候就在想的事.txt
data/newsgroup/儿童诗歌.txt
data/newsgroup/一千零一夜.txt
data/newsgroup/._newsgroup.txt
Traceback (most recent call last):
  File "gen.py", line 287, in <module>
    main(args.viz)
  File "gen.py", line 134, in main
    RV3 = RendererV3(DATA_PATH,max_time=SECS_PER_IMG)
  File "/data/home/xuhuaren/ft_local/SynthText_Chinese_version-master/synthgen.py", line 369, in __init__
    self.text_renderer = tu.RenderFont(data_dir)
  File "/data/home/xuhuaren/ft_local/SynthText_Chinese_version-master/text_utils.py", line 112, in __init__
    fn=osp.join(data_dir,'newsgroup/'))
  File "/data/home/xuhuaren/ft_local/SynthText_Chinese_version-master/text_utils.py", line 547, in __init__
    line=line.decode('utf-8')
  File "/data/home/xuhuaren/anaconda3/envs/py2.7/lib/python2.7/encodings/utf_8.py", line 16, in decode
    return codecs.utf_8_decode(input, errors, True)
UnicodeDecodeError: 'utf8' codec can't decode byte 0xab in position 45: invalid start byte
